src/scss/_type.scss

```diff
@@ -1,4 +1,4 @@
-@use './vars' as *;
+@import './vars';
```

Comment:
Should we migrate to `dart-sass` and always do `@use`?  I was reading something online about that being a best practice

Reply:
We are migrating in phillips-public but until that happens we have to use imports.